interface/asset-manager.md

```diff
@@ -0,0 +1,42 @@
+# Asset Manager
+
+## Overview 
+The **Asset Manager** is an organized collection of **Assets** that can be readily created, imported, deleted and updated. Below, you will find a depiction of the **Asset Manager** and its features.
+
+![](../.gitbook/assets/interface/asset-manager/asset-manager.PNG) 
+
+## Views in the Assert Manager 
+The **Assert Manager** can be *toggled* between a `flat` or `folder` `tree` `view` by clicking on the *stripe icon* ![](../.gitbook/assets/interface/asset-manager/stripeIcon.PNG) located in the toolbar of the **Asset Manager**.
+
+The `flat tree` `view` shows **Assets** in a single-panel window while the `folder tree view` shows **Assets** in a dual-panel window. 
```

Comment:
```suggestion
The `flat` view shows the **Assets** of the current folder in a single-panel, while the `folder tree` view shows both the folder's **Assets** and a side panel to navigate to other folders. 
```

Reply:
On the next line this explanation is given
I am not sure if this is needed?